in_toto/in_toto_mock.py

```diff
@@ -31,8 +31,21 @@
   in-toto-mock --name write-code -- touch foo.py
```

Comment:
I think you need to specify the `.py` extension with `in-toto-mock`. Actually, should that be `in_toto_mock.py`!?

Also, is the `--` by itself needed/correct in `-- touch foo.py`?

I was expecting it to be: `$ in-toto-mock.py --name write-code touch foo.py`

Reply:
Nope, [`in-toto-mock` works fine](https://github.com/in-toto/in-toto/blob/develop/setup.py#L80-L85) when in-toto is installed.

Reply:
Per *nix and [argparse](https://docs.python.org/dev/library/argparse.html#arguments-containing) convention is used to separate optional arguments from positional arguments.